pkg/ddevapp/commands.go

```diff
@@ -1,47 +1,48 @@
 package ddevapp
 
 import (
-	"github.com/ddev/ddev/pkg/fileutil"
-	"github.com/ddev/ddev/pkg/globalconfig"
-	"github.com/ddev/ddev/pkg/util"
-	copy2 "github.com/otiai10/copy"
+	"fmt"
 	"os"
+	"path"
 	"path/filepath"
+
+	dockerImages "github.com/ddev/ddev/pkg/docker"
+	"github.com/ddev/ddev/pkg/dockerutil"
+	"github.com/ddev/ddev/pkg/globalconfig"
+	"github.com/ddev/ddev/pkg/nodeps"
+	"github.com/ddev/ddev/pkg/util"
 )
 
-// PopulateCustomCommandFiles sets up the custom command files in the project
+// PopulateGlobalCustomCommandFiles sets up the custom command files in the project
 // directories where they need to go.
-func PopulateCustomCommandFiles(app *DdevApp) error {
-
+func PopulateGlobalCustomCommandFiles() error {
```

Comment:
This rename isn't really necessary, but it just makes it more explicit that this is only touching **global** commands. I initially thought it did something with project-level commands as well before I read through the code.